.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ repos:
 
       hooks:
         - id: go-fmt
-        - id: go-vet
+#        - id: go-vet #TODO: figure out: a solution of build constraints exclude all
```

Comment:
What's the issue here sorry?

Reply:
The issue was with the integration tests living in its own directory/package